config/src/config/transaction_filter_type.rs

```diff
@@ -0,0 +1,192 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
```

Comment:
nit: given this is a new file, we don't need to mention this line about Meta (same below). 😄 But, doesn't matter too much!